lib/host.js

```diff
@@ -1,20 +1,30 @@
 var _ = require('lodash-contrib');
 var request = require('request');
+var futils = require('./fhutils');
 
-module.exports = function host(cb) {
-  var url = 'https://' + process.env.FH_MILLICORE + '/box/srv/1.1/ide/apps/app/hosts';
-  var data = {
-    "guid": process.env.FH_INSTANCE,
-    "env": process.env.FH_ENV
+module.exports = function(cfg) {
+  if (!cfg) {
+    return;
```

Comment:
@sedroche looking at the other modules, they either don't care, or have this ugly piece of code, or use `assert.ok(cfg, 'cfg is undefined');` Does this assert approach looks better to you ?